docker/dev_env/run.sh

```diff
@@ -0,0 +1,79 @@
+#! /usr/bin/env bash
+
+set -e
+
+# Kudos https://stackoverflow.com/a/10910180 for macOS group ID reading
+case "$OSTYPE" in
+linux*)
+  host_docker_gid="$(getent group docker | cut -d: -f3)"
+  ;;
+
+darwin*)
+  host_docker_gid="$(dscl . -read /Groups/docker | awk '($1 == "PrimaryGroupID:") { print $2 }')"
+  ;;
+
+*)
+  printf "'%s' is not supported for Openverse development. Only Linux and macOS are supported. Use WSL if on Windows." "$OSTYPE" >>/dev/stderr
+  exit 1
+  ;;
+esac
+
+suppress_output() {
+  "$@" 2>/dev/null 1>/dev/null
+}
+
+opener_home="/home/opener"
+
+run_args=(
+  -i
+  --rm
```

Comment:
The issue is if you don't --rm, then you need to variably `exec` instead of `run`. So we'd have to `docker ps` in `run.sh` to check to see if the container is already available, and if not, then use `run` instead. I wonder if that actually improves start up speed?

Reply:
Maybe not. Let's leave it like this for now.